spring-components/rdf4j-spring-demo/pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ ******************************************************************************
+  ~ Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.
```

Comment:
No real need to put the header in an xml file, to be honest.

Reply:
ok, removed (from properties file, too)